scripts/create-user.sh

```diff
@@ -11,22 +11,24 @@ source "${SCRIPT_DIR}/common.sh"
 DEFAULT_SPACE=sandbox
 FROM_ADDRESS='gov-uk-paas-support@digital.cabinet-office.gov.uk'
 # shellcheck disable=SC2016
-SUBJECT='Welcome to the Government PaaS'
+SUBJECT='Welcome to GOV.UK PaaS'
 # shellcheck disable=SC2016,SC1078
 MESSAGE='Hello,
 
-Your account for the Government PaaS service has been created.
+Your account for GOV.UK PaaS is ready:
 
-Your organisation is \"${ORG}\" and your login and password are:
+ - username: ${EMAIL}
+ - organisation: ${ORG}
 
- - login: ${EMAIL}
- - password: ${PASSWORD}
+Please use this link to activate your account and set a password. The link will only work once:
+${INVITE_URL}
+
+You can find advice about choosing a password:
+https://docs.cloud.service.gov.uk/#choosing-passwords
 
 To get started, look at our Quick Setup Guide:
 https://docs.cloud.service.gov.uk/#quick-setup-guide
 
-You should make sure to change your password, as explained in the Quick Setup Guide.
-
 You can find our privacy policy here:
 https://docs.cloud.service.gov.uk/#privacy-policy
 
```

Comment:
After "Regards" suggest

GOV.UK PaaS team.

PS Some departmental email systems will check links in inbound emails as part of their virus protection. This may have invalidated your one-time link. If this is the case please contact support to set your password another way.